### DISCUSSION TITLE
Errors: JSON.parse: unexpected character at line

### DISCUSSION THREAD

**Discussion by lochbika** [rern/rAudio#294] [Fixed Issues]
I activated the relays plugin and get this when I want to open the system settings page:

`Errors: JSON.parse: unexpected character at line 1 column 1164 of the JSON data▮{ "cpuload" : "0.20 0.32 0.36" , "cputemp" : 45.1 , "startup" : "6s (kernel) + 13s (userspace) " , "throttled" : "0x0" , "time" : "17:43:26 2021-04-06" , "timezone" : "Europe/Amsterdam" , "uptime" : "6h 58m" , "uptimesince" : "2021-04-06 10:44" , "audioaplayname" : "hifiberry-dacplus" , "audiooutput" : "HiFiBerry DAC+ Pro" , "bluetooth" : false , "btdiscoverable" : false , "btformat" : false , "hostapd" : false , "hostname" : "rAudio" , "kernel" : "5.10.25-5-ARCH armv7l" , "lcd" : false , "lcdchar" : false , "lcdcharaddr" : "" , "lcdcharconf" : "" , "list" : [{"icon":"microsd","mountpoint":"/","mounted":true,"source":"/dev/mmcblk0p2","size":"4.8GB/30GB"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/ROOT","mounted":true,"source":"/dev/sda1","size":"3.2GB/4.9GB"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/intData","mounted":true,"source":"/dev/sda2","size":"61GB/105GB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/podcasts","mounted":true,"source":"//192.168.1.71/podcasts","size":"3.2TB/3.6TB"}] , "ntp" : "pool.ntp.org" , "powerbutton" : false , "powerbuttonconf" : "40 33" , "reboot" : "" , "regdom" : "00" , "relays" : false , "relayspins" : , "rpi01" : false , "rpimodel" : "Raspberry Pi 3 Model B Rev 1.2" , "soc" : "BCM2837" , "soccpu" : "Cortex-A53" , "socram" : "867MiB" , "socspeed" : "1200" , "soundprofile" : false , "version" : "1" , "versionui" : 20210403 , "wlan" : true , "soundprofileval" : "18000000 60 1500 1000"}`

**Comment by rern** [comment]
Something's wrong with `/etc/relays.conf`
Reset the file:
```sh
echo '{
  "name": {
    "11": "DAC",
    "13": "PreAmp",
    "15": "Amp",
    "16": "Subwoofer"
  },
  "on": {
    "on1": 11,
    "ond1": 2,
    "on2": 13,
    "ond2": 2,
    "on3": 15,
    "ond3": 2,
    "on4": 16
  },
  "off": {
    "off1": 16,
    "offd1": 2,
    "off2": 15,
    "offd2": 2,
    "off3": 13,
    "offd3": 2,
    "off4": 11
  },
  "timer": 5
}
' > /etc/relays.conf
```
Then try again.

**Reply by lochbika** [reply]
Thanks! That worked :+1:
